### PR TITLE
Improve Ref Docs

### PR DESCRIPTION
**Pull request by sommersoft**
[Per Issue #566](https://github.com/adafruit/circuitpython/issues/566)

Didn't rename any of the examples on this one. Felt it better to leave them all separated, although the case could be made to consider `accel.py` as the simpletest.

If any examples are renamed, this Learn Guide will need to be updated:
https://learn.adafruit.com/circuitpython-hardware-lis3dh-accelerometer?view=all#software

### CODE REVIEW

File: adafruit_lis3dh.py [hunk 1]
```diff
@@ -204,45 +224,50 @@ def read_adc_mV(self, adc): # pylint: disable=invalid-name
...
-            :param int threshold: A threshold for the tap detection.  The higher the value
-              the less sensitive the detection.  This changes based on the accelerometer
-              range.  Good values are 5-10 for 16G, 10-20 for 8G, 20-40 for 4G, and 40-80 for
-              2G.
-            :param int time_limit: TIME_LIMIT register value (default 10).
-            :param int time_latency: TIME_LATENCY register value (default 20).
-            :param int time_window: TIME_WINDOW register value (default 255).
-            :param int click_cfg: CLICK_CFG register value."""
+        """
+        Set the tap detection parameters.
```
Comment (kattni) [inline adafruit_lis3dh.py]: Change to `The tap detection parameters.`

**Review by kattni — CHANGES_REQUESTED** [review]
Great job! I suggested one change in-line. I agree that `accel.py` should be renamed to the appropriate simpletest. Please make these changes and this should be good to go!

**Comment by sommersoft** [comment]
Updates made. Learn Guide will need to be changed.

**Review by kattni — APPROVED** [review]
Looks great, thanks!

**Comment by kattni** [comment]
Guide updated.

**Pull request merged**